lib/strings.nix

```diff
@@ -1467,6 +1467,73 @@ rec {
       [ "\"" "'" "<" ">" "&" ]
       [ "&quot;" "&apos;" "&lt;" "&gt;" "&amp;" ];
 
+  /**
+    Quotes a string `s` so that it can be embedded in a Python script as a [bytes
+    literal](https://docs.python.org/3/reference/lexical_analysis.html#bytes-literals).
+
+    This function allows you to turn a Nix string value into a Python bytes value rather than a
+    Python string value. This is because the Nix expression language and the Python programming
+    language use the term “string” to refer to different things. In the Nix expression language,
```

Comment:
What happened here?

Reply:
OK, so when I look at that file locally, I can see that there are Unicode smart quotes around the word string. When I looked at that file on GitHub, it was showing mojibake for me, but then I refreshed the page, and the mojibake went away. Does it look right for you if you refresh the page?